src/tigris.ts

```diff
@@ -119,13 +121,14 @@ const DEST_NAME_KEY = "destination-name";
 export class Tigris {
 	private readonly grpcClient: TigrisClient;
 	private readonly observabilityClient: ObservabilityClient;
-	private readonly config: TigrisClientConfig;
+	private readonly _config: TigrisClientConfig;
 
 	/**
 	 *
 	 * @param  {TigrisClientConfig} config configuration
 	 */
 	constructor(config?: TigrisClientConfig) {
+		dotenv.config();
```

Comment:
Will this override existing variables in `process.env`? Basically Next runtime would already have loaded a configuration in `process.env`. Or should we only load if `config === undefined && process.env.TIGRIS_URI === undefined`?

Reply:
nvm.

https://github.com/motdotla/dotenv#what-happens-to-environment-variables-that-were-already-set